src/css/profile/mobile/common/textinput.less

```diff
@@ -103,7 +103,7 @@ textarea.ui-text-input {
 	box-sizing: border-box;
 	display: block;
 	width: 100%;
-	line-height: 54 * @unit_base;
+	line-height: 26 * @sp_base;
```

Comment:
Do we use the @sp_base for the 'line-height' not @px_base?

Reply:
yes, in some places of guide I see font size values marked with sp so I assume all guide values refer to sp

Reply:
@HunseopJeong you are right. For Line height we should use px